src/sentry/api/decorators.py

```diff
@@ -5,12 +5,13 @@
 from django.http import HttpResponse
 from functools import wraps
 
-from sentry.models import ApiKey
+from sentry.models import ApiKey, ApiToken
 
 
 def is_considered_sudo(request):
     return request.is_sudo() or \
-        isinstance(request.auth, ApiKey)
+        isinstance(request.auth, ApiKey) or \
```

Comment:
No good reason. Originally because i did `(isinstance(request.auth, ApiToken) && request.auth.user is None)` but that does not actually work :D